src/app/base/countWeight.js

```diff
@@ -0,0 +1,27 @@
+import { range } from './../constants/constants';
+
+// Count weight category by BMI index
+const countWeight = (container, indexBmi) => {
+    const bmi = container.querySelector('.js-index-bmi');
+    const category = container.querySelector('.js-weight-category');
+
+    bmi.textContent = indexBmi;
+
+    if (indexBmi < 15 || indexBmi > 25) {
```

Comment:
The values 15, 25 etc. would be better as a constant. Firstly you use it in more places and the code would be more understandable, because now I don't really know what the numbers mean